### ISSUE TITLE
Add a new organization to ROR: 
 
Jagiellonian College in Toruń

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#582]
Summary of request: Add a new organization to ROR

Update record:
Name of organization: 
ROR ID: 
Which part of the record needs to be changed? 
Description of change: 

Merge/split/deprecate records:
Organizations affected by this change:
ROR ID(s) of organization(s) affected by this change: 
How should the record(s) be changed? 

Add record:
Name of organization: Jagiellonian College in Toruń
Website: www.fves.eu
Link to publications: https://www.fves.eu/en/journal-archiv.html
Organization type: Education (A university or similar institution involved in providing education and educating/employing researchers)
Wikipedia page: https://en.wikipedia.org/wiki/Jagiellonian_College
Wikidata ID: 
ISNI ID: 
GRID ID: 
Crossref Funder ID: 
Aliases: Vysoká škola Jagiellońská v Toruni
Labels:
Acronym/abbreviation: FVES
Related organizations: 
City: Uherské Hradiště (CZ)
Country: Czech Republic
Geonames ID:
Year established: 2003
How will a ROR ID for this organization be used? To identify affiliations for research publications, To identify affiliations for datasets, To acknowledge research funders, To acknowledge a research facility, To be integrated in a scholarly publishing system

Other information about this request: You can contact me on jan.vrba@fves.eu for more details. Thank you.

**Comment by adambuttrick** [comment]
Educational institute in the Czech Republic. Publishes journal _International Journal of Public Administration, Management and Economic Development_. Hosts academic conferences. Limited publication affiliation usage, but otherwise appears in scope for ROR.

Publication affiliation usage: https://bibliotekanauki.pl/articles/438648 ;

**Comment by adambuttrick** [comment]
@arthurpsmith Good catch. I'm going to treat this as a request for the main campus and create a separate issue to discuss whether the Czech institute is in scope as a distinct entity. Since the page lists it as "part of the Jagiellonian University in Toruń, Poland," there needs to be sussing out of whether any of the affiliation usage references the Czech vs Polish departments.
